### DISCUSSION TITLE
Accesso a Hassio da HTTPS non funzionante

### DISCUSSION THREAD

**Discussion by Moxser** [indomus/forum#665] [HUB: Home Assistant]
Scusatemi se apro un altra discussione.
Oggi vanno tutte storte.
Dopo un ripristino di uno snapshot di ieri non riesco più ad accedere dall'esterno a Hassio.
Duck e Nginx sono configurati correttamente e funzionanti.
La porta 443 sul fritz è ok.
Tutto funzionava bene fino a 1 ora fa.
Ora https://ilmiohassio.duckdns.org non funziona mi da sempre "400: Bad Request"

Questo è l'errore che ho nei log 
Received X-Forwarded-For header from an untrusted proxy 172.30.33.2

Come posso risolvere?
Scusatemi e capisco che probabilmente è una domanda fatta 1000 volte ma non so che pesci pigliare.

Gtazie a tutti

**Comment by Moxser** [comment]
Ho aggiunto 172.30.33.2 alla configurazione e ora funziona tutto. Ma è normale secondo voi?

**Comment by snakuzzo** [comment]
Leggi [QUI](https://indomus.it/news/aggiornamento-2021-7-per-home-assistant-core/)

P.S. cerca di rimanere sempre al passo...ela prossima volta cerca sia sul sito che qui su discussions...se ne è parlato tante volte

**Reply by Moxser** [reply]
Hai ragione Diego. 
Grazie mille
